contracts/proxy/CMakeLists.txt

```diff
@@ -2,7 +2,7 @@ file(GLOB ABI_FILES "*.abi")
 configure_file("${ABI_FILES}" "${CMAKE_CURRENT_BINARY_DIR}" COPYONLY)
 
 add_wast_executable(TARGET proxy
-  INCLUDE_FOLDERS "${STANDARD_INCLUDE_FOLDERS}"
-  LIBRARIES eosiolib
+  INCLUDE_FOLDERS "${STANDARD_INCLUDE_FOLDERS}" ${Boost_INCLUDE_DIR} 
```

Comment:
This seems to be a result of including the `eosio.system` contract's include so that we can process EOS tokens in the proxy.  In order to keep dependencies on contracts low, maybe we should separate out the definition of the EOS currency from the rest of the system contract so that we don't have to pull in things like Boost HANA at build time just to reference the currency?